### PR TITLE
Change the default font family to Arial

### PR DESCRIPTION
**Pull request by LinhNguyen2901**

### CODE REVIEW

**Review by itskarelleh — CHANGES_REQUESTED** [review]
Do a Google search on adding fallback font family to Expo + React Native.

We don't want to change the default font we just simply want to have a second font family in case the user does not have this one installed on their device.

**Comment by LinhNguyen2901** [comment]
I have fixed it to add Arial as the fallback font instead of changing the default font.

File: lesson-authoring/package.json
Comment (itskarelleh) [inline lesson-authoring/package.json:1]: I'm not sure why this file was modified for this PR

**Comment by itskarelleh** [comment]
@LinhNguyen2901 Your commit had a lot of changes to the config files which I had to delete but also deleted them on the main branch. So I reverted the changes.

**Pull request merged**